starfish/image/_filter/mean_high_pass.py

```diff
@@ -58,27 +49,25 @@ def add_arguments(cls, group_parser: argparse.ArgumentParser) -> None:
             help="indicates that the image stack should be filtered in 3d")
 
     @staticmethod
-    def high_pass(image: np.ndarray, size: Number) -> np.ndarray:
+    def high_pass(
+            image: Union[xr.DataArray, np.ndarray], size: Union[Number, Tuple[Number]]
+    ) -> Union[xr.DataArray, np.ndarray]:
```

Comment:
The output type for this method appears to only be `nd.ndarray`.  I can't see any paths that would yield a `xr.DataArray`.

Reply:
Good point. Fixed.